packages/ui/src/components/content/ContentListPanel.vue

```diff
@@ -0,0 +1,93 @@
+<script setup lang="ts" generic="T">
+import { ref, computed } from 'vue'
+import type { Ref } from 'vue'
+import Checkbox from '../base/Checkbox.vue'
+import ContentListItem from './ContentListItem.vue'
+import { type ContentItem } from './ContentListItem.vue'
+import { DropdownIcon } from '@modrinth/assets'
+// @ts-ignore
+import { RecycleScroller } from 'vue-virtual-scroller'
```

Comment:
The vue-virtual-scroller library causes a serious bug in WebKit that freezes the entire app on some devices (https://github.com/modrinth/code/issues/1127). This makes most of the app unusable, as I can’t access my projects. 

To fix it, please switch to another virtual list library like `vue-typed-virtual-list`. I’ve used it before (example: https://github.com/modrinth/code/commit/797e1f1f214011cc36112f88bde7c609237fb2e1), though it had some issues with logs not attaching to the bottom. However, this shouldn’t affect the projects list.

Reply:
Will fix before release